frontend/src/hooks/api/aliases.ts

```diff
@@ -138,16 +139,8 @@ export function getAllAliases(
   return (randomAliases as AliasData[]).concat(customAliases);
 }
 
-export function getFullAddress(alias: AliasData, profile: ProfileData) {
-  if (!isRandomAlias(alias) && typeof profile.subdomain === "string") {
-    return `${alias.address}@${profile.subdomain}.mozmail.com`;
-  }
-  if (alias.domain === (1 as Domain_RelayFirefoxCom)) {
-    // 1 = @relay.firefox.com
-    return `${alias.address}@relay.firefox.com`;
-  }
-  // 2 = @mozmail.com
-  return `${alias.address}@mozmail.com`;
+export function getFullAddress(alias: AliasData) {
+  return alias.full_address;
```

Comment:
Other reviewers: `alias` gained `full_address` months ago in 3.1.10, https://github.com/mozilla/fx-private-relay/pull/1425, and replaces the hard-coded addresses only valid in prod.